RudderAnalytics/Request/BlockingRequestHandler.cs

```diff
@@ -173,24 +173,24 @@ public async Task MakeRequest(Batch batch)
                 var requestData = Encoding.UTF8.GetBytes(json);
 
                 // Compress request data if compression is set
-//                 if (_client.Config.Gzip)
-//                 {
-// #if NET35
-//                     _httpClient.Headers.Set(HttpRequestHeader.ContentEncoding, "gzip");
-// #else
-//                     //_httpClient.DefaultRequestHeaders.Add("Content-Encoding", "gzip");
-// #endif
-//
-//                     // Compress request data with GZip
-//                     using (MemoryStream memory = new MemoryStream())
-//                     {
-//                         using (GZipStream gzip = new GZipStream(memory, CompressionMode.Compress, true))
-//                         {
-//                             gzip.Write(requestData, 0, requestData.Length);
-//                         }
-//                         requestData = memory.ToArray();
-//                     }
-//                 }
+                if (_client.Config.Gzip)
+                {
+#if NET35
+                     _httpClient.Headers.Set(HttpRequestHeader.ContentEncoding, "gzip");
+#else
+                    // _httpClient.DefaultRequestHeaders.Add("Content-Encoding", "gzip");
```

Comment:
yes, because we are setting it here: https://github.com/rudderlabs/rudder-sdk-.net/blob/d232575236d531560b9416919cedbe2337d896ea/RudderAnalytics/Request/BlockingRequestHandler.cs#L280